packages/babel-preset-default/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ### Breaking Change
 
 - Removed `babel-core` dependency acting as Babel 7 bridge ([#13922](https://github.com/WordPress/gutenberg/pull/13922). Ensure all references to `babel-core` are replaced with `@babel/core` .
-- Preset updated to include `@wordpress/babel-plugin-import-jsx-pragma` plugin integration ([#13540](https://github.com/WordPress/gutenberg/pull/13540)).
+- Preset updated to include `@wordpress/babel-plugin-import-jsx-pragma` plugin integration ([#13540](https://github.com/WordPress/gutenberg/pull/13540)). It should no longer be explicitly included in your Babel config.
```

Comment:
This change is nested inside section: 4.0.0 (2019-03-06). Should we create a new 4.1.0 (Unreleased) section so this breaking change refers to the next version of the package?

Reply:
It was already published as 4.0.0. This PR only adds a note which should help prevent the issue @swissspidy and @ocean90 encountered when upgrading.